db/recovery_db.go

```diff
@@ -98,7 +98,7 @@ func (db *RecoveryDB) GetContractSwapInfo(id types.Hash) (*EthereumSwapInfo, err
 }
 
 // PutSwapPrivateKey stores the given ephemeral swap private key share for the given swap ID.
-func (db *RecoveryDB) PutSwapPrivateKey(id types.Hash, sk *mcrypto.PrivateSpendKey, env common.Environment) error {
+func (db *RecoveryDB) PutSwapPrivateKey(id types.Hash, sk *mcrypto.PrivateSpendKey) error {
 	val, err := json.Marshal(sk.Hex())
 	if err != nil {
 		return err
```

Comment:
I added some code [in this PR](https://github.com/AthanorLabs/atomic-swap/pull/239) that will let you marshal/unmarshal `sk` directly. Calling `json.Marshal(sk.Hex())` is equivalent to `[]byte(fmt.Sprintf("%q", sk.Hex()))` if you don't want to check the error. Just pointing that out, not saying it should be one way or the other.